daml-lf/spec/daml-lf-1.rst

```diff
@@ -1921,19 +1921,19 @@ need to be evaluated further. ::
    ——————————————————————————————————————————————————— ValScenario
      ⊢ᵥ  s
 
-   ——————————————————————————————————————————————————— ValUnBondedMathContext
+   ——————————————————————————————————————————————————— ValUnboundedMathContext
      ⊢ᵥ  LitRoundingMode
 
 
                            ┌────────┐
    Update Values           │ ⊢ᵥᵤ  u │
                            └────────┘
 
-     ⊢ᵥ  e
+     ⊢ᵥᵤ  e
    ——————————————————————————————————————————————————— ValUpdatePure
      ⊢ᵥᵤ  'pure' @τ e
 
-     ⊢ᵥ  e₁
+     ⊢ᵥᵤ  e₁
```

Comment:
problably better like that, but not that thanks to `ValUpdate` this was actually correct as well.

Reply:
> For well-typed terms, the two are equivalent. But I find it easier to understand if we require an update value here rather than an arbitrary value.

In this case I agree.